src/goto-bmc/goto_bmc_parse_options.cpp

```diff
@@ -48,7 +48,7 @@ int goto_bmc_parse_optionst::doit()
   }
 
   if(
-    cmdline.args.size() != 1 ||
+    cmdline.args.size() > 1 &&
```

Comment:
This change appears to be in the wrong commit, given that the commit title refers to regression tests, but this change is a change to the implementation.

Reply:
This appears to be resolved.